backend/src/test/java/com/sim_backend/state/StateTest.Java

```diff
@@ -0,0 +1,44 @@
+package com.sim_backend;
+
+import static org.junit.jupiter.api.Assertions.fail;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.*;
+
+import io.javalin.Javalin;
+import io.javalin.plugin.bundled.CorsPluginConfig;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.*;
+import com.sim_backend.state.*;
+
+public class StateTest {
+
+    private SimulatorStateMachine testStateMachine;
+
+    @BeforeEach
+    void setUp() {
+        testStateMachine = new SimulatorStateMachine();     
+    }
+
+    @Test
+    // Initialize test
+    void testInit() throws Exception{
+        try{
```

Comment:
We do not need to wrap `assert` in a try catch block. The test framework will handle it and put the appropriate output in the console.